"0","df <- data %>% "
"0","  select(!(starts_with(""p6"")|starts_with(""p13"")|starts_with(""p7"")|starts_with(""d40"")|starts_with(""qa3"")|starts_with(""qa2"")))"
"0",""
"0","var <- sapply(df, var) %>% "
"0","  as.list() %>% "
"0","  .[. != 0 | is.na(.)] %>% "
"0","  names()"
"2","Warning: NAs introduced by coercion"
"2","Warning: NAs introduced by coercion"
"2","Warning: NAs introduced by coercion"
"2","Warning: NAs introduced by coercion"
"0","df <- df %>% "
"0","  select(all_of(var)) %>% "
"0","  select(!c(""doi"", ""version"", ""caseid"", ""uniqid"", ""serialid"")) %>% "
"0","  mutate(across(cat, haven::as_factor)) %>% "
"0","  sjlabelled::remove_all_labels()"
"1","[1m[33mError[39m in `mutate()`:[22m
[38;5;254m[36mâ„¹[38;5;254m In argument: `across(cat, haven::as_factor)`.[39m
[1mCaused by error in `across()`:[22m
[33m![39m Can't select columns that don't exist.
[31mâœ–[39m Column `cat` doesn't exist.
[90mRun `]8;;x-r-run:rlang::last_trace()rlang::last_trace()]8;;` to see where the error occurred.[39m
"
